index.html

```diff
@@ -3913,7 +3913,8 @@ <h2><span class="chunk">cLLi</span> Content Light Level Information</h2>
              
           <p>MaxFALL (Maximum Frame Average Light Level) indicates the maximum value of the frame 
              average light level (in cd/m2 or nits) of the entire playback sequence. MaxFALL is 
-             calculated by averaging the decoded luminance values of all the pixels within a frame.</p>
+	     calculated by first averaging the decoded luminance values of all the pixels in each frame,
+             and then using the value for the frame with the highest value</p>
```

Comment:
The wording change looks good. There's a missing period at the end of the sentence (and also at the end of the following paragraph that describes MaxCLL).